src/calculateRentalCost.js

```diff
@@ -1,10 +1,16 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
+
+  if (days >= 7) {
```

Comment:
Prefer `if` with `return` over `if else` to simplify later conditions. You can return the total cost immediately after applying the discount for long-term rentals (checklist item 4).